src/plugins/github/graphql.js

```diff
@@ -324,7 +326,7 @@ function* continuationsFromIssue(
   path: $ReadOnlyArray<string | number>
 ): Iterator<Continuation> {
   const b = build;
-  if (result.comments.pageInfo.hasNextPage) {
+  if (result.comments && result.comments.pageInfo.hasNextPage) {
```

Comment:
Curious whether your tests would have caught the absence of this change?

Reply:
I think so.